package.json

```diff
@@ -8,8 +8,8 @@
     "test": "jest",
     "build": "tsc",
     "lint": "eslint --ext .ts src/",
-    "start": "npx ts-node src/index.ts",
-    "start:dev": "npx nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.test.ts' --exec 'npx ts-node' src/index.ts"
+    "start": "ts-node src/index.ts",
+    "start:dev": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.test.ts' --exec 'npx ts-node' src/index.ts"
```

Comment:
Perhaps add `ts-node` to dev deps as well? Does using it without npx have an impact?

Reply:
I'm a little fuzzy here since my mental rule for dev deps is "do I need this running on production server?" and since you're using it here in the `start` script my thoughts were "yes, this dep goes in regular dependencies".

Either way, I found it weird using ts-node in production. Normally you would want to transpile all your TS files and use something like pm2 or forever in production with your index.js file.

On the npx question, is a tool for not having to install global dependencies in your machine. It works great in cases like create-react-app but not as part of your package.json

Reply:
> On the npx question, is a tool for not having to install global dependencies in your machine. It works great in cases like create-react-app but not as part of your package.json

I meant asking if npx has an impact on the performance, I do know npx, which is why I used it here hehe. 

I agree that ts-node on production is wrong though. Let’s leave that as is for now and I’ll change it after, unless you feel comfortable also changing it on this PR.